RuriLib/Blocks/Parsing/Methods.cs

```diff
@@ -112,9 +113,9 @@ public static string QueryJsonToken(BotData data, string json, string jToken, st
 
         #region REGEX
         public static List<string> MatchRegexGroupsRecursive(BotData data, string input,
-            string pattern, string outputFormat, string prefix = "", string suffix = "")
+            string pattern, string outputFormat, bool multiLine, string prefix = "", string suffix = "")
```

Comment:
If you add parameters in the middle it will break all previous configs that were using the old signature and make OB2 versions incompatible. Only add parameters at the end and with a default value specified otherwise the signature changes and we will have problems.

NOTE: In this case, since it's a special block, when transpiling the method to C# it would be verbose to add parameters after prefix and suffix so I suggest adding a new method with the old signature for compatibility, and make it call the new method with RegexOptions.None directly ^^

Reply:
I solved it in a different way. I added a default value to the function
```suggestion
            string pattern, string outputFormat, bool multiLine = false, string prefix = "", string suffix = "")
```
Then, where C# code is generated, I made it so it would write multiLine as a named argument, so the result is this:
```
var parseOutput = MatchRegexGroups(data, data.SOURCE.AsString(), "^123[a-z]{4}$", "[0]", multiLine: false, "", "");
```
I've tested it by loading a config from version without multiline support and it works without problems.